packages/scipy/patches/disable_scipy_stats_mvn.patch

```diff
@@ -0,0 +1,39 @@
+commit d6740caa7400105a578ace74f8c25bbb3434178c
+Author: Roman Yurchak <rth.yurchak@pm.me>
+Date:   Wed Oct 10 14:47:12 2018 +0200
+
+    Disable scipy.stats.mvn
```

Comment:
Possibly an f2c issue, more details in https://github.com/iodide-project/pyodide/pull/211#issuecomment-428928156